gadget.Dockerfile

```diff
@@ -20,6 +29,16 @@ RUN cd /gadget && go mod download
 COPY ./ /gadget
 RUN cd /gadget/gadget-container && make gadget-container-deps
 
+# BTFGen
+
+COPY --from=objs /objs /objs
+RUN set -ex; mkdir -p /tmp/btfs && \
+	if [ "$ENABLE_BTFGEN" = true ]; then \
+		cd /gadget && \
+		./tools/getbtfhub.sh && \
+		LIBBPFTOOLS=/objs BTFHUB=/tmp/btfhub ./tools/btfgen.sh; \
+	fi
+
```

Comment:
Yeah, I think it'll help when building the think locally, however I'm not sure if this will mess up with the Github Actions cache as [it's limit is 5GB](https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy) but we're using more than it. :thinking: